templates/borg-backup.sh.epp

```diff
@@ -160,13 +161,15 @@ backup_borg() {
   fi
 
   borg create "${options[@]}" --stats --verbose "$dst::backup-$(date "+%Y%m%d-%H%M%S")" "$src"
+<% if ($manage_prune == true) { -%>
```

Comment:
you can shorten this to:
```suggestion
<% if $manage_prune { -%>
```

Reply:
See https://github.com/voxpupuli/puppet-borg/pull/137/files#diff-fb441c2a865ae587ac8c8a6f2b324f1414a3821d099b31c5b5a077533ef0dad9L20, I just referenced the syntax there. Shorted them all.